client/src/Layout/DashboardLayout.tsx

```diff
@@ -22,17 +30,28 @@ function DashboardLayout({ component }: Props): JSX.Element {
 
   return (
     <Box className={classes.root}>
-      <AppBar position="relative" color="inherit" elevation={6}>
+      <AppBar position="sticky" color="inherit" elevation={6} className={clsx(isHomePage && classes.HomePage)}>
         <Toolbar>
           <Box className={classes.logo}>
             <img src={Logo} alt="Logo Image" />
           </Box>
           {!loggedInUser ? (
             <Box className={classes.wrapper}>
-              <Button component={Link} to={'#'} color="inherit" className={classes.member}>
+              <Button
+                component={Link}
+                to={'#'}
```

Comment:
No URL here?

Reply:
I noticed that too. I think @DejanKatovic left it that way because the page it points to is yet to be implemented